### ISSUE TITLE
Add a screen-reader only Metadata h2

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/dpul-collections#832]
### Summary or User Story

Users using a screen reader weren't sure they were reading the metadata - the screen reader should announce it, and jumping between headings should allow them to get to the metadata.

### Acceptance Criteria

- [ ] There's an invisible "Metadata" h2 that a screen reader will read.

### First step

Add it. Ensure it's translated.
